scrapers/historical.js

```diff
@@ -216,9 +216,51 @@ const getHistoricalAllDataV2 = (data) => {
 	};
 };
 
+const getHistoricalUSADataV2 = async (keys, redis) => {
+	let casesResponse;
+	let deathsResponse;
+	try {
+		casesResponse = await axios.get(`${base}time_series_covid19_confirmed_US.csv`);
+		deathsResponse = await axios.get(`${base}time_series_covid19_deaths_US.csv`);
+	} catch (err) {
+		console.log({
+			message: 'has been ocurred an error in JHUhistorical USA REQUEST',
+			errno: err.errno,
+			url: err.config.url
+		});
+		return;
+	}
+	const parsedCases = await parseCsvData(casesResponse.data);
+	const parsedDeaths = await parseCsvData(deathsResponse.data);
+	const timelineKey = Object.keys(parsedCases[0]).splice(11);
+	const usaCountryData = countryUtils.getCountryData('USA');
+	const result = Array(parsedCases.length).fill({}).map((_, index) => {
```

Comment:
could you also quickly refactor all scrapers to map results instead of having an results array and pushing to it in a foreach

Reply:
sure